administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -63,35 +203,48 @@ function triggerSave() {
 		height: 1500px;
 		border: 0 !important;
 	}
+
+	.reference-side h3 {
+		float: left;
+		width: 50%;
+	}
 ');
+
+$input      = $this->app->input;
+$layout     = $input->get('layout', '', 'string');
+$aComponent = $input->get('acomponent', '', 'string');
+$aView      = $input->get('aview', '', 'string');
+$fLanguage  = $input->get('forcedlanguage', '', 'string');
+
 ?>
-<button id="toogle-left-panel">Show/Hide Reference (PoC)</button>
+<button id="toogle-left-panel" class="btn btn-small">Show/Hide Reference (PoC)</button>
```

Comment:
I'm thinking of treating this in another PR (JS code revision) where I optimize my JS code and that Hide Show will be dynamic:
When reference is shown : Hide Reference
When referece is hidden : Show Reference